From a335dbad1f371c4c8783361485f770ca2b4a946a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kai=20K=C3=B6hne?= <kai.koehne@qt.io>
Date: Thu, 16 Oct 2025 10:20:23 +0200
Subject: [PATCH 226/239] Doc: Remove Details link from qmlformat documentation

There's no Details section for qmlformat, so it ends up actually
linking to the Details section of qmlpreview ...

Change-Id: Ib6142b362b15a72ef847bb6e751cf64bd64a9e04
Reviewed-by: Fabian Kosmale <fabian.kosmale@qt.io>
(cherry picked from commit 9ad53a4fd20d3a7aa4174cb8ac8fe4fde9e09589)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qml/doc/src/tools/qtqml-tooling-qmlformat.qdoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/doc/src/tools/qtqml-tooling-qmlformat.qdoc b/src/qml/doc/src/tools/qtqml-tooling-qmlformat.qdoc
index 1c19cf256b..d8a011a1b0 100644
--- a/src/qml/doc/src/tools/qtqml-tooling-qmlformat.qdoc
+++ b/src/qml/doc/src/tools/qtqml-tooling-qmlformat.qdoc
@@ -10,7 +10,7 @@
 
 \section1 qmlformat
 \e qmlformat is a tool that automatically formats QML files in accordance
-with the \l{QML Coding Conventions}. \l{Details}{More...}
+with the \l{QML Coding Conventions}.
 
 \table
 \header
-- 
2.51.2

